pkg/controller/route/route_controller.go

```diff
@@ -128,6 +128,7 @@ func (r *ReconcileRoute) Reconcile(request reconcile.Request) (reconcile.Result,
 	// if absent then create
 	routeCreated, ok := instance.ObjectMeta.Annotations[`routeCreated`]
 	if !ok {
+		instance.ObjectMeta.Annotations = make(map[string]string) // This keeps the controller from crashing when annotation later on
```

Comment:
Please move this line to just before the Annotations are used. And the comment at the end is not needed.

Reply:
Done.